infoview/index.tsx

```diff
@@ -94,7 +94,7 @@ function Main(props: {}) {
                 const isCursor = locationEq(loc,curLoc.loc);
                 return <Info loc={loc} isPaused={paused} setPaused={setPause(i)} key={i} isPinned={true} isCursor={isCursor} onEdit={onEdit} onPin={unpin(i)}/>}) }
             {!isPinned(curLoc.loc) && <Info loc={curLoc.loc} isPaused={curLoc.paused} setPaused={setPause()} key={pinnedLocs.length} isPinned={false} isCursor={true} onEdit={onEdit} onPin={pin}/>}
-            <details>
+            <details open={!config.infoViewAutoOpenShowGoal}>
```

Comment:
Is the `<Messages/>` here rendered always, even if collapsed?  Is this a performance issue?

Reply:
Yes it is rendered always. Afaik it's not a perf issue.